maintainers/maintainer-list.nix

```diff
@@ -3168,6 +3168,12 @@
       fingerprint = "389A 78CB CD88 5E0C 4701  DEB9 FD42 C7D0 D414 94C8";
     }];
   };
+  dukc = {
+    email = "ajieskola@gmail.com";
+    github = "dukc";
+    githubId = 24233408;
+    name = "Ate Eskola";
+  };
```

Comment:
split this in to a separate commit before the main commit called `maintainers: add dukc`

Reply:
I split into separate commit, but it ended up after, not before, the main commit. I tried to swap their order once, screwed up somehow and had to do it again.

Reply:
Adding yourself as a maintainer to dmd should be in a separate commit from maintainers list one

I recommend learning git rebase

Reply:
Ok tried again, better luck this time.